src/App.tsx

```diff
@@ -1,37 +1,50 @@
 import { useState } from "react";
 import "./styles/App.css";
 import SurvivorSelection from "./components/SurvivorSelection";
-import { Survivor, UserSelection } from "./utilities/types";
+import { Items, Survivor, UserSelection } from "./utilities/types";
 import Instructions from "./components/Instructions";
 import ItemSelection from "./components/ItemSelection";
 
 //app level: rendering the individual steps of the flow and nothing else
 function App() {
   const [userSelection, setUserSelection] = useState<UserSelection>(
-    {} as UserSelection
+    {userID: 0,
+    userSurvivor: {} as Survivor,
+    userItems: []
+  }
   ); //*to store survivor selected by user, user id, and items selected by user
 
+  const [numberOfItemsSelected, setNumberOfItemsSelected] = useState<number>(0)
```

Comment:
not sure you really need this..

userSelection.userItems.length()

Reply:
updated logic to be in MultiShopSelection component under useEffect